### ISSUE TITLE
Questions about MOB data

### ISSUE THREAD

**Issue by jincanke** [YingMa1993/CARD#21]
Congratulate!  I have a question, in the MOB dataset, why do you only select neuron cells to deconvolve?  Actually, this isn't a simulated dataset, it's true. Do you think it will affect the results of deconvolution, just as spots would be occupied by immune or non-neuron cells?

**Comment by YingMa0107** [comment]
Hi @jincanke, 

Thank you very much for your interest in our method. 

We used the neuron cells for deconvolution is for the evaluation purpose as each layer is dominant by their specific neurons and this would allow as to calculate the ARI between the dominant cell types inferred layer and the H&E image inferred layer, also this will allows us to evaluate the robustness of different methods when there might be missing cell types in the scRNAseq reference in the real data, just like what we evaluated in one simulation setting. The layered structure here we used is also according to the original publication of this ST dataset [https://www.science.org/doi/10.1126/science.aaf2403](url) where it mainly focused on the major layers. 

In general, for the minor cell types, it might not affect the results of deconvolution too much. For the major cell types, it will.
